pkg/types/automation/interfaces.go

```diff
@@ -48,10 +48,16 @@ type Encoder interface {
 
 type LogEventProvider interface {
 	GetLatestPayloads(context.Context) ([]UpkeepPayload, error)
+	SetConfig(LogEventProviderConfig)
 	Start(context.Context) error
 	Close() error
 }
 
+type LogEventProviderConfig struct {
+	NumLogUpkeeps    uint32
+	FastExecLogsHigh uint32
```

Comment:
missing `FastExecLogsLow`

Reply:
i see this PR is merged but we might need changes based on comments on https://github.com/smartcontractkit/chainlink-automation/pull/314/files